Editorial updates. Added steps to link with Runtime Manager. (#988)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/mule-standalone.adoc

endif::[]
:keywords: mule, studio, enterprise, ee, premium features, paid features, purchase, license, licensed

> If you want to host Mule runtime engine in your own environment, xref:hardware-and-software-requirements.adoc[verify that you meet hardware and software requirements], then xref:runtime-installation-task.adoc[download and install Mule].

Reviewer: Always include "and" if you use ", then" (which is not, by itself, a real transition, unless you use "...; then, ...").
```suggestion
If you want to host Mule runtime engine (Mule) in your own environment, first xref:hardware-and-software-requirements.adoc[verify that you meet all hardware and software requirements], and then xref:runtime-installation-task.adoc[download and install Mule].
```

You don't need to install Mule in the following scenarios:

> * When you use Studio for development and testing, because Studio contains Mule.

Reviewer 2: because Studio contains Mule _embedded._

> * When you use Studio for development and testing, because Studio contains Mule embedded.

Reviewer: You don't really need the second half
```suggestion
* When you use Anypoint Studio for development and testing
```
But if you want it, consider
```suggestion
* When you use Anypoint Studio for development and testing, because Studio contains Mule
```
OR
```suggestion
* When you use Anypoint Studio (which contains Mule) for development and testing
```

> * When you deploy applications to CloudHub or Runtime Fabric.

Reviewer: ```suggestion
* When you deploy applications to CloudHub or Anypoint Runtime Fabric
```

> If you want to manage your on-premises Mule instance using Runtime Manager, xref:runtime-manager::servers-create.adoc[register it using the Runtime Manager agent]

Reviewer: period
```suggestion
If you want to manage your on-premises Mule instance using Runtime Manager, xref:runtime-manager::servers-create.adoc[register it using the Runtime Manager agent].
```
